entropylab/tests/test_complex_db.py

```diff
@@ -0,0 +1,60 @@
+from entropylab import Script, Graph
+from entropylab.api.data_reader import ExperimentReader
+from entropylab.api.execution import EntropyContext
+from entropylab.graph_experiment import PyNode
+from entropylab.instruments.lab_topology import ExperimentResources, LabResources
+from entropylab.results_backend.sqlalchemy.db import SqlAlchemyDB
+import numpy as np 
+import subprocess
+
+db_file=r'C:\repos\entropy\entropylab\tests\myproj'
```

Comment:
If I may add:
1, Don't use an absolute path as this will fail on other people's machines.
2. Put the project under a `tests_cache` folder. That way it won't be added to source control.